configs/setups/awiesm3/awiesm3.yaml

```diff
@@ -0,0 +1,783 @@
+        
+        #proc_list_entry: '[str(x) for x in range(1,${nproc}+1)]'
+        nproc: 128
+        proc_list_entry: '[str(x) for x in range(1,${nproc}+1)]' 
+        proc_list: $(( ${proc_list_entry} ))
+
+        run_year: ${start_date!syear}
+        run_month: 1
+        run_day: 1
+        runtime_seconds: "$(( ${next_date} - ${start_date} ))"
+        ndays_per_run: "$(( ${runtime_seconds} / 86400 ))"
+        TM5: 0
+        fourxCO2: f
+        onepcCO2: f
+        fixed_CO2: -1
+        fixed_ndep: -1
+        fixed_LU: -1
+   
+        forcing_sources:                
+                soil_layer_type: ${lpj_guess.input_dir}/slt/slt_${mygrid}.nc
+                tmp_rc_file: ${lpj_guess.input_dir}/lpjg_steps.rc.j2
+                CO2: ${lpj_guess.input_dir}/cmip6_forcing/mole_fraction_of_carbon_dioxide_in_air_input4MIPs_lpjg.nc
+                dir_landuse: ${lpj_guess.input_dir}/dirmaker
+                dir_fire: ${lpj_guess.input_dir}/dirmaker
+                ins_files: ${namelist_dir}/*.ins
+                #ece_gridlist: ${lpj_guess.input_dir}/gridlists/gridlist_TL159_CORE2.txt #is this the correct file for ece_gridlist.txt?
+                ece_gridlist: ${lpj_guess.input_dir}/gridlists/gridlist_TL255_CORE2.txt #is this the correct file for ece_gridlist.txt? 
+                lpjg_cmip6_output_ins: ${namelist_dir}/lpjg_default_cmip6_output.ins
+                "all_ins_[[proc_list-->PROC_NUM]]": ${namelist_dir}/*.ins 
+                "soil_layer_type_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/slt/slt_${mygrid}.nc
+                "tmp_rc_file_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/lpjg_steps.rc.j2
+                "CO2_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/cmip6_forcing/mole_fraction_of_carbon_dioxide_in_air_input4MIPs_lpjg.nc
+                "guess_ins_[[proc_list-->PROC_NUM]]": ${namelist_dir}/guess.ins.j2
+                "ecearth_ins_[[proc_list-->PROC_NUM]]": ${namelist_dir}/ecearth.ins
+                "globsoil_ins_[[proc_list-->PROC_NUM]]": ${namelist_dir}/global_soiln.ins
+                "lpjg_cmip6_output_ins_[[proc_list-->PROC_NUM]]": ${namelist_dir}/lpjg_default_cmip6_output.ins
+                #"ece_gridlist_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/gridlists/gridlist_TL159_CORE2.txt #is this the correct file for ece_gridlist.txt?
+                "ece_gridlist_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/gridlists/gridlist_TL255_CORE2.txt #is this the correct file for ece_gridlist.txt?
+                temp_namcouple: /home/a/a270270/temp_confiles/namcouple 
+                "mock_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/dirmaker
+
+        forcing_in_work:
+                soil_layer_type: slt_${mygrid}.nc
+                tmp_rc_file: lpjg_steps.rc
+                CO2: mole_fraction_of_carbon_dioxide_in_air_input4MIPs_lpjg.nc
+                temp_namcouple: namcouple
+                dir_landuse: /landuse/dirmaker
```

Comment:
@hajlaci 

In this issue (https://github.com/esm-tools/esm_tools/issues/1283#issuecomment-2701374424) a better solution was give, e.g.

Please, grep for the dirmakers, delete those lines and add:

```yaml
    create_folders:
        - ${work_dir}/landuse
        - ${work_dir}/fire
        - ...
```